src/DistributedArrays.jl

```diff
@@ -487,4 +487,21 @@ Base.scale!(A::DArray, x::Number) = begin
     return A
 end
 
+Base.sum(d::DArray) = reduce(Base.AddFun(), d)
+Base.maximum(d::DArray) = reduce(Base.MaxFun(), d)
+Base.minimum(d::DArray) = reduce(Base.MinFun(), d)
+Base.maxabs(d::DArray) = maximum([fetch(r) for r in [@spawnat p maxabs(localpart(d)) for p in procs(d)]])
```

Comment:
Ah. [This](https://github.com/JuliaParallel/DistributedArrays.jl/blob/1eef9381917858271088e8d21639fec09e48aa02/src/DistributedArrays.jl#L426) signature is too restrictive. It excludes the functors so it falls back on the method in Base. If the `Functions` is removed, I think it should work.